Main/Main.css

```diff
@@ -6,8 +6,11 @@
     border: 0px;
 }
 :root {
-    --colour-primary : #2ecc71;
-    --colour-secondary: white;
+    --color-primary : #2ecc71;
+    --color-secondary: white;
+    --color-primary-bright:rgb(57,255,20);
+    --color-black:#000000bf;
+    --color-grey :grey;
     --alert-success:#60C060;

```

Comment:
Here also, specify which color is secondary, primary bright, etc.

Reply:
I have update this thank you for the suggestion